signingscript/script.py

```diff
@@ -99,6 +103,7 @@ def get_default_config(base_dir=None):
         'dmg': 'dmg',
         'hfsplus': 'hfsplus',
         'gpg_pubkey': None,
+        'widevine_cert': None,
```

Comment:
Hm. Should we have a cert per signature level? (dep/nightly/prod). Or is the assumption that

- there is a prod cert and a dep cert
- depsigning is a separate pool from prod signing
- so we can have a single cert per pool

Or am I misreading what the cert is?

Reply:
We currently only have two: dep, and prod.  The cert here is the public side of the keypair that corresponds to what is in the HSM. I think this is analogous to the GPG pubkey?

Reply:
Ok. The current layout, where we have a single cert per script, means we can't have both dep and prod widevine signing on the same instance. That's probably an ok assumption since we have a depsigning pool.

Reply:
I was assuming this was going to be our model given https://bugzilla.mozilla.org/show_bug.cgi?id=1540277#c4

Reply:
Ah. We currently support multiple levels per scriptworker instance, by specifying config in `passwords.json` for each supported cert level. However, passwords.json is not currently flexible in its layout, so a widevine cert would at best be another tuple entry.

If this becomes an issue, we can make the cert config a dictionary, and specify level to cert. We don't have to do that now.